src/pages/Models/Details/_model-details.scss

```diff
@@ -1,7 +1,7 @@
 .model-details {
   display: grid;
   grid-template-columns: 350px 1fr;
-  min-height: 100vh;
+  min-height: calc(100vh - 48px); // hack to fix initial set of 100vh
```

Comment:
Is this working around a vanilla issue? If so, could this be upstreamed?

Reply:
The site header will always be 48px high so this is how I'd style it for now.

Later down the line, when we have a batter idea of how other templates will look, we may/could refactor it to use CSS grid. However, fine for now. (Pls remove notes/comments referring to it as a "hack" before landing.)

Reply:
Thanks, comment about the hack removed, added a comment as to where `48px` comes from.